static/javascript/index.js

```diff
@@ -1,5 +1,112 @@
+    if (response.status === 'connected') {
+      // Logged into your app and Facebook
+        
+      testAPI(response);
+    } else if (response.status === 'not_authorized') {
+      // The person is logged into Facebook, but not your app.
+      document.getElementById('status').innerHTML = 'Please log ' +
+        'into this app.';
+    } else {
+      // The person is not logged into Facebook, so we're not sure if
+      // they are logged into this app or not.
+      
+      FB.getLoginStatus(function(response) {
+        $.ajax({
+            url: '/log_out',
+            type: 'DELETE',
+            success: sign_out
+        });
+            console.log("In getLoginStatus");
+
+        });
+
+      document.getElementById('status').innerHTML = 'Please log ' +
+        'into Facebook.'
+
+
+    }
+}
+
+
+// This function is called when someone finishes with the Login
+// Button.  See the onlogin handler attached to it in the sample
+// code below.
+function checkLoginState() {
+    FB.getLoginStatus(function(response) {
+      statusChangeCallback(response);
+    });
+}
+
+
+window.fbAsyncInit = function() {
+FB.init({
+    appId      : '{app_id}',
+    cookie     : true,  // enable cookies to allow the server to access 
+                        // the session
+    xfbml      : true,  // parse social plugins on this page
+    version    : 'v2.8' // use graph api version 2.8
+});
+
+// Now that we've initialized the JavaScript SDK, we call 
+// FB.getLoginStatus().  This function gets the state of the
+// person visiting this page and can return one of three states to
+// the callback you provide.  They can be:
+//
+// 1. Logged into your app ('connected')
+// 2. Logged into Facebook, but not your app ('not_authorized')
+// 3. Not logged into Facebook and can't tell if they are logged into
+//    your app or not.
+//
+// These three cases are handled in the callback function.
+
+// FB.getLoginStatus(function(response) {
+//     statusChangeCallback(response);
+// });
+
+// The below punctution is needed to contain something to avoid Syntax error
+};
+
+// Here we run a very simple test of the Graph API after login is
+// successful.  See statusChangeCallback() for when this call is made.
+function postRequest(){
+    console.log("In post request");
+     window.location = "/";
+}
+
+function testAPI(response) {
+    console.log('Welcome!  Fetching your information.... ');
+
+    /// TODO TAKE CARE OF THIS SECTION  SEEEK postREQUEST
+
+    var accessToken = response['authResponse']['accessToken'];
+    console.log('THIS IS THE ACCESS TOKEN');
+    console.log(accessToken);
+    var data = {"accessToken": accessToken}
+    $.get("/log_in", data, postRequest);
+    console.log("After POST Request");
+
+
+    FB.api('/me', function(response) {
+      console.log('Successful login for: ' + response.name);
+      document.getElementById('status').innerHTML =
+        'Thanks for logging in, ' + response.name + '!';
+    });
```

Comment:
Yes it prints next to the sign-in/logout button. Kind of like a little flash message of success.